src/Locations/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Locations from './Locations'
+import {Route} from 'react-router-dom'
+
+export default function index() {
+    return (
+        <div>
+            <Route path='/' component={Locations}/>
+        </div>
+    )
+}
```

Comment:
Is this file necessary? Because the routing logic is pretty simple, I'd recommend just creating a file named `src/Locations.js` or `src/components/Locations.js`